Cargo.toml

```diff
@@ -5,3 +5,7 @@ authors = ["Bilal Elmoussaoui <bil.elmoussaoui@gmail.com>", "Julian Hofer"]
 edition = "2018"
 
 [dependencies]
+isahc = "0.9"
+quick-xml = "0.18"
+thiserror = "1.0"
+mockito = "0.25.2"
```

Comment:
I tend to avoid using enforcing micro releases ^^

Reply:
Actually me as well, that wasn't done on purpose :)

Reply:
Fixed